### PR TITLE
RESTful API: define specific action for the API.

### PR DESCRIPTION
**Pull request by jerome-benoit**
+ Lower case only,
+ More standard auth enpoints name.

That will lead to duplication during the transition.

Signed-off-by: Jérôme Benoit <jerome.benoit@sap.com>

### CODE REVIEW

File: src/types/Server.ts [hunk 1]
```diff
@@ -368,4 +368,12 @@ export enum ServerAction {
   HTTP_REQUEST = 'HttpRequest',
   HTTP_RESPONSE = 'HttpResponse',
   HTTP_ERROR = 'HttpError',
+
+  // RESTful API
+  SIGNIN = 'signin',
+  SIGNON = 'signon',
+  SIGNOUT = 'signout',
+  RESTful_PASSWORD_RESET = 'passwordreset',
```
Comment (LucasBrazi06) [inline src/types/Server.ts:376]: RESTful_PASSWORD_RESET => REST_PASSWORD_RESET or RESTFUL_PASSWORD_RESET
All in uppercase.

File: src/server/rest/router/GlobalRouter.ts [hunk 2]
```diff
@@ -7,14 +7,17 @@ import { utilRouter } from './util/UtilRouter';
 
 export const globalRouter = express.Router();
 
-// API
-globalRouter.use('/v1/api', AuthService.authenticate(), tenantRouter);
+// API version 1 routes
+const currentVersion = 1;
+const currentVersionBaseURI = `/v${currentVersion}`;
```
Comment (LucasBrazi06) [inline src/server/rest/router/GlobalRouter.ts:12]: If we add a directory v1, no need to add a variable here I think

File: src/server/rest/router/util/UtilRouter.ts [hunk 3]
```diff
@@ -1,10 +1,11 @@
 import express, { NextFunction, Request, Response } from 'express';
 
+import { ServerAction } from '../../../../types/Server';
 import { StatusCodes } from 'http-status-codes';
 
 export const utilRouter = express.Router();
 
-utilRouter.get('/ping', async (req: Request, res: Response, next: NextFunction) => {
+utilRouter.get('/' + ServerAction.RESTful_PING, async (req: Request, res: Response, next: NextFunction) => {
```
Comment (LucasBrazi06) [inline src/server/rest/router/util/UtilRouter.ts:8]: '/' + ServerAction.RESTful_PING => `/${ServerAction.RESTFUL_PING}` ? :-)

File: src/types/Server.ts (hunk 1 above)
Comment (jerome-benoit) [reply src/types/Server.ts]: Done.

File: src/server/rest/router/GlobalRouter.ts (hunk 2 above)
Comment (jerome-benoit) [reply src/server/rest/router/GlobalRouter.ts:12]: You will not have the directory name mapping into the router out of box, unless you do some directory listing and parsing. It's better to just use a constant.

File: src/server/rest/router/util/UtilRouter.ts (hunk 3 above)
Comment (jerome-benoit) [reply src/server/rest/router/util/UtilRouter.ts]: Done.

**Pull request merged**